content/pt/docs/concepts/cluster-administration/kubelet-garbage-collection.md

```diff
@@ -0,0 +1,71 @@
+---
+reviewers:
+title: Configurando o Garbage Collection do kubelet
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+
+O Garbage collection(Coleta de lixo) é uma função útil do kubelet que limpa imagens e recipientes não utilizados. O kubelet executará o garbage collection para contêineres a cada minuto e para imagens a cada cinco minutos.
+
+Ferramentas externas de garbage collection não são recomendadas, pois podem potencialmente interromper o comportamento do kubelet removendo os recipientes que existem.
```

Comment:
maybe change to:

"o comportamento do kubelet removendo os `contêineres ` que existem."

Reply:
@femrtnz you can use the suggestion feature, example:

```suggestion
Ferramentas externas de garbage collection não são recomendadas, pois podem potencialmente interromper o comportamento do kubelet removendo os contêineres que existem.
```

so your contributions in Code Review are stored in the PR's commits history ;)